ci-operator/step-registry/cucushift/installer/rehearse/aws/ipi/private/shared-vpc-phz/sts/provision/cucushift-installer-rehearse-aws-ipi-private-shared-vpc-phz-sts-provision-chain.yaml

```diff
@@ -3,6 +3,7 @@ chain:
   steps:
   - ref: ipi-conf
   - ref: ipi-conf-telemetry
+  - ref: aws-provision-iam-user-minimal-permission
```

Comment:
I am thinking of creating some dedicated minimal install workflow to run the testing instead of update the existing ones.